### DISCUSSION TITLE
[Bug]: API Logout response 500

### DISCUSSION THREAD

**Discussion by du-nong-0070** [danny-avila/LibreChat#3303] [Troubleshooting]
### What happened?

API Logout response 500

### Steps to Reproduce

1. login
2. logout

### Relevant log output

```shell
The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received null
```

### Screenshots

![Screenshot 2024-07-09 at 17 33 12](https://github.com/danny-avila/LibreChat/assets/90369101/a91868b4-c3e9-4ed1-9d7c-412f292ff2be)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
the issue is not clear on how to reproduce

**Reply by du-nong-0070** [reply]
Just start logging and then log out.
And the response of the logout API is 500.
Here is step to reproduce

https://github.com/user-attachments/assets/a8f0de6f-2b58-45cd-9f3b-f195b194166c
